src/algebraic_geometry/Scheme.lean

```diff
@@ -84,6 +85,32 @@ Schemes are a full subcategory of locally ringed spaces.
 instance : category Scheme :=
 induced_category.category Scheme.to_LocallyRingedSpace
 
+/--
+The global sections, notated Gamma.
+-/
+def Γ : Schemeᵒᵖ ⥤ CommRing :=
+(induced_functor Scheme.to_LocallyRingedSpace).op ⋙ LocallyRingedSpace.Γ
+
+lemma Γ_def : Γ = (induced_functor Scheme.to_LocallyRingedSpace).op ⋙ LocallyRingedSpace.Γ := rfl
```

Comment:
I'm not particularly complaining, but what is the motivation to define this at all?

Reply:
To eliminate the need of unfolding.

Reply:
See:
https://github.com/leanprover-community/mathlib/blob/dd43823c00c0814d676fd3b90fbf790d81a15cfd/src/algebra/geom_sum.lean#L24-L25

https://github.com/leanprover-community/mathlib/blob/dd43823c00c0814d676fd3b90fbf790d81a15cfd/src/algebra/geom_sum.lean#L41-L42

https://github.com/leanprover-community/mathlib/blob/dd43823c00c0814d676fd3b90fbf790d81a15cfd/src/analysis/normed_space/hahn_banach.lean#L52-L54

https://github.com/leanprover-community/mathlib/blob/dd43823c00c0814d676fd3b90fbf790d81a15cfd/src/topology/uniform_space/basic.lean#L1227-L1228

https://github.com/leanprover-community/mathlib/blob/dd43823c00c0814d676fd3b90fbf790d81a15cfd/src/data/polynomial/eval.lean#L228

https://github.com/leanprover-community/mathlib/blob/dd43823c00c0814d676fd3b90fbf790d81a15cfd/src/analysis/asymptotics.lean#L104-L105

https://github.com/leanprover-community/mathlib/blob/dd43823c00c0814d676fd3b90fbf790d81a15cfd/src/data/array/lemmas.lean#L51-L52

https://github.com/leanprover-community/mathlib/blob/dd43823c00c0814d676fd3b90fbf790d81a15cfd/src/data/list/basic.lean#L3269

https://github.com/leanprover-community/mathlib/blob/dd43823c00c0814d676fd3b90fbf790d81a15cfd/src/data/nat/dist.lean#L17

https://github.com/leanprover-community/mathlib/blob/dd43823c00c0814d676fd3b90fbf790d81a15cfd/src/data/polynomial/degree/basic.lean#L52

https://github.com/leanprover-community/mathlib/blob/dd43823c00c0814d676fd3b90fbf790d81a15cfd/src/linear_algebra/smodeq.lean#L28-L29

Reply:
Okay!